### PR TITLE
Fundamentals diagnostic response

### PR DESCRIPTION
**Pull request by kenvng**

### CODE REVIEW

File: diagnostic.md [hunk 1]
```diff
@@ -12,17 +12,19 @@ Graces-MacBook-Pro:wdi grace$ pwd
 Graces-MacBook-Pro:wdi grace$ mkdir try
 Graces-MacBook-Pro:wdi grace$ cd try
 Graces-MacBook-Pro:try grace$
-```
+
 
 What is the full path to the current directory?
 
 _Replace this text with your answer._
+/Users/grace/wdi
```
Comment (GA-MEB) [inline diagnostic.md:20]: Nope, we went inside the `try` directory. The path is now `/Users/grace/wdi/try`

File: diagnostic.md [hunk 2]
```diff
@@ -12,17 +12,19 @@ Graces-MacBook-Pro:wdi grace$ pwd
...
 
 ## Question 2
 
 Describe GitHub, **briefly** in your own words.
 
 _Replace this text with your answer._
+files sharing and saving, like a database
```
Comment (GA-MEB) [inline diagnostic.md:27]: Sort of. It's not as much for sharing files as it is for sharing Git projects.

File: diagnostic.md [hunk 3]
```diff
@@ -46,13 +49,14 @@ while (x < 5) {
 How many times does the loop run?  What's the value of `x` after the while loop?
 
 _Replace this text with your answer._
+4
```
Comment (GA-MEB) [inline diagnostic.md:52]: Nope, sorry. Try working this one out by hand - play the role of the computer and execute the program line by line.

File: diagnostic.md [hunk 4]
```diff
@@ -63,6 +67,9 @@ var result = add(0, -5);
 What is the value of result?  Explain your answer.
 
 _Replace this text with your answer._
+-5
+function strangeAdd assigning the value of a and b.
+if a > 0 or b > 0, either case is true, both not true so a + b, 0+(-5)=-5
```
Comment (GA-MEB) [inline diagnostic.md:72]: Not correct. See notes on Q4.

File: diagnostic.md [hunk 5]
```diff
@@ -94,3 +102,6 @@ var teamMember = {
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of `teamMember['name']['surname']`?
 
 _Replace this text with your answer._
+value of teamMember hobby is exercise
```
Comment (MattB-GA-Boston) [inline diagnostic.md]: Yes.

File: diagnostic.md [hunk 6]
```diff
@@ -94,3 +102,6 @@ var teamMember = {
...
+value of teamMember name is JasonWeeks
+value of teamMember name surname is JasonWeeks Weeks
```
Comment (GA-MEB) [inline diagnostic.md:107]: Correct, incorrect, incorrect.

**Comment by GA-MEB** [comment]
3/7

**Pull request closed without merging**